packages/flutter_tools/test/commands.shard/hermetic/analyze_test.dart

```diff
@@ -158,18 +156,19 @@ void main() {
         'startColumn': 4,
       },
       'message': 'Prefer final for variable declarations if they are not reassigned.',
+      'code': 'var foo = 123;',
```

Comment:
what caused this diff?

Reply:
`WrittenError.fromJson` asserts that the code key is not null. This test doesn't care about that, so I just had to add something arbitrary here.